cidc_ngs_pipeline_api/output_API.schema.json

```diff
@@ -4,6 +4,9 @@
   "title": "Pipeline output_API.JSON schema",
   "type": "object",
   "description": "Schema for pipeline's output_API.JSONs",
+  "properties": {
+    "id": {"type":"null"}
```

Comment:
What's the purpose of this?

Reply:
so that output_APIs cannot have "id" as an entry, as that is unclear if that's 'CIMAC id' or 'run id', etc
forgot to put that in the commit msg as well!